modules/mobile/spec/workers/pre_cache_appointments_job_spec.rb

```diff
@@ -31,48 +31,50 @@
     context 'with no errors' do
       it 'caches the expected appointments' do
         VCR.use_cassette('appointments/get_facilities', match_requests_on: %i[method uri]) do
-          VCR.use_cassette('appointments/get_cc_appointments_default', match_requests_on: %i[method uri]) do
-            VCR.use_cassette('appointments/get_appointments_default', match_requests_on: %i[method uri]) do
-              expect(Mobile::V0::Appointment.get_cached(user)).to be_nil
-              subject.perform(user.uuid)
+          VCR.use_cassette('appointments/get_mfs_facilities', match_requests_on: %i[method uri]) do
```

Comment:
Its surprisingly using it... the test was failing without the new VCR because it was also getting the new url and didn't know how to handle it.

Reply:
Any idea why it's being used without the feature flag being enabled? Seems like it shouldn't. Also, is the `'appointments/get_facilities'` VCR above still necessary? I don't understand why both would be called. Maybe I'm missing a piece and it's being called for reasons other than the appointments proxy.

Reply:
Ok, I get it now. The issue is that all feature flags are on by default in specs. You disable them in the list specs but not here. IMO what you should do is have two versions of this test in this file, one with the feature flag on and the other with the feature flag off. This test as it's written does not actually use this vcr: 
`VCR.use_cassette('appointments/get_facilities', match_requests_on: %i[method uri]) do`

Reply:
I would also suggest removing this from line 110:
`VCR.use_cassette('appointments/get_facilities', match_requests_on: %i[method uri]) do`
It doesn't use that cassette because the error happens before it fetches facilities. Now that we're using multiple facilities endpoints, it just adds confusion and will probably just become cruft once we permanently switch to the new approach.

Reply:
Oh ok I see. So now theres one that tests using `'get_facilities'` and another that tests using `'get_mfs_facilities'`